release-doc-in-process/all_vf.TTL

```diff
@@ -618,6 +618,22 @@ vf:conformsTo
         vs:term_status      "unstable" ;
         rdfs:comment        "The primary resource knowledge specification or definition of an existing or potential resource." .
 
+vf:stage
+        a                   owl:ObjectProperty ;
+        rdfs:label          "stage" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:RecipeFlow) ] ;
```

Comment:
For RecipeFlows and Commitments to use stage requirements, they need to know the stages of EconomicResources, altho the information can be derived from their events.